pkg/liqo-controller-manager/virtualNode-controller/namespaceMap_lifecycle.go

```diff
@@ -0,0 +1,115 @@
+package virtualNode_controller
+
+import (
+	"context"
+	"fmt"
+	mapsv1alpha1 "github.com/liqotech/liqo/apis/virtualKubelet/v1alpha1"
+	const_ctrl "github.com/liqotech/liqo/pkg/liqo-controller-manager"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/util/slice"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// remove Finalizer and Update the NamespaceMap
+func (r *VirtualNodeReconciler) deleteNamespaceMap(nm mapsv1alpha1.NamespaceMap) error {
+
+	nm.SetFinalizers(slice.RemoveString(nm.GetFinalizers(), namespaceMapFinalizer, nil))
+	klog.Info("Someone try to delete namespaceMap, ok delete!!")
```

Comment:
Two things:
- I would avoid using multiple !! in the log message
- I would always put the "object" of the action (e.g.; the name of the deleting namespaceMap )

Reply:
Yes you are right, thank you